cfgov/jinja2/v1/_includes/organisms/table.html

```diff
@@ -12,35 +12,44 @@
 
    ========================================================================== #}
 
+{% if value.data  %}
 <table class="o-table
               {{ 'o-table__row-links' if value.row_links else '' }}
-              table__stack-on-small
-              table__entry-header-on-small">
+              {{ 'table__stack-on-small' if value.is_stacked else '' }}
+              {{ 'table__striped' if value.is_striped else '' }}
+              {{ 'u-w100pct' if value.is_full_width else '' }}">
+    {% set row_index = 0 %}
+    {% if value.first_row_is_table_header and value.data | list | length > 0 %}
+    {% set row_index = 1 %}
     <thead>
+        {# Check to verify the row isn't empty #}
+        {% if value.data[0] | reject('none') | list | length %}
         <tr>
-            {% for thead in value.headers %}
-                <th>{{ thead }}</th>
+            {% for cell in value.data[0] %}
+            <th scope="col">
+                {{ cell if cell is not none else '' | trim | safe | linebreaksbr }}
+            </th>
             {% endfor %}
         </tr>
+        {% endif %}
     </thead>
-    {% for row in value.rows %}
+    {% endif %}
+    <tbody>
+    {% for row in value.data[row_index:] or [] %}
+        {% if row | reject('none') | list | length %}
         <tr>
-        {% for cell in row %}
-            {% set colcount = loop.index %}
-            {% for thead in value.headers %}
-                {% if colcount == loop.index %}
-                    {% if 'rich_text_blob' in cell.block_type %}
-                        <td data-label="{{ thead }}">
-                            {{ parse_links(cell.value) | safe }}
-                        </td>
-                    {% else %}
-                        <td data-label="{{ thead }}">
-                            {{ cell | safe }}
-                        </td>
-                    {% endif %}
+            {% for cell in row or [] %}
+                {% set tag = 'th scope="row"' if value.first_col_is_header and loop.first else 'td' %}
+                {% if value.data[0][loop.index0] is not none %}
+                   {% set data_label = 'data-label="{0}"'.format(value.data[0][loop.index0] | striptags | trim) %}
```

Comment:
Should the data-label only be added if `value.first_row_is_table_header`? Otherwise, the first row is used as a header and also displayed when the table stacks on mobile:

<img width="810" alt="screen shot 2016-09-22 at 2 47 30 pm" src="https://cloud.githubusercontent.com/assets/778171/18767655/5b5057be-80d5-11e6-9c9d-8833bdc90955.png">
<img width="431" alt="screen shot 2016-09-22 at 2 47 45 pm" src="https://cloud.githubusercontent.com/assets/778171/18767665/65941c7e-80d5-11e6-9c4b-06fd6d114458.png">